lib/src/server_messages.dart

```diff
@@ -270,3 +283,11 @@ class ErrorField {
     }
   }
 }
+
+class Notification {
+  final int processId;
```

Comment:
Also, is it possible to have some way of having the connection know if the notification was triggered by that connection? Would allow listeners to say 'Oh, I sent this, I can ignore it' but would also allow listeners to structure their app so they can listen for their own notifications, too.